frontend/UserList.vue

```diff
@@ -172,6 +196,63 @@ export default {
         limit: 40
       }
     },
+    updateUserRoleModal: function (role) {
+      if (!!this.disableConfirmButton) {
```

Comment:
I don't think the `!!` is necessary here?

Reply:
As long as the component initializes this property, then we wouldn't need it. I can change it to a simple check.